src/control/lib/hardware/sysfs/provider.go

```diff
@@ -484,7 +484,7 @@ func (s *Provider) ibStateToNetDevState(stateStr string) hardware.NetDevState {
 	stateSubstrs := strings.Split(string(stateStr), ": ")
 	stateNum, err := strconv.Atoi(stateSubstrs[0])
 	if err != nil {
-		s.log.Debugf("unable to parse IB state %q: %s", stateStr, err.Error())
+		s.log.Tracef("unable to parse IB state %q: %s", stateStr, err.Error())
```

Comment:
This might be worth pushing up to a higher notification level. I would not expect this to happen.

Reply:
Should it be an ERROR-level message?

Reply:
Notice or error. It's not fatal, so maybe Notice would be a better fit.

Reply:
Changed it to Notice.